model/src/main/java/javasabr/mqtt/model/topic/AbstractTopic.java

```diff
@@ -78,4 +82,16 @@ protected static int countOccurrencesOf(String str, String sub) {
     }
     return count;
   }
+
+  protected static String replaceToConstant(String segment) {
```

Comment:
Could we rename it to `replaceToConstantIfRequired` or `replaceWildcardToConstant` ?